chanzuckerberg/edu-design-system
# fix(typography): add missing eds-font-size-20

### Summary:

- this size was referenced in code, but not defined as a token or variable.
- add it in after cross-checking with the related design
- Verify the sizing logic (1rem = 16px/16px) for font-size 20px

### Test Plan:


  - verify breakpoint appears properly
  - verify the token and css variable is defined
